ci/prepare.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -e
+
+export ERLANG_VERSION="17.1"
+export ELIXIR_VERSION="v1.0.4"
+
+# If you have a elixir_buildpack.config, do this instead:
+#export ERLANG_VERSION=$(cat elixir_buildpack.config | grep erlang_version | tr "=" " " | awk '{ print $2 }')
+#export ELIXIR_VERSION=v$(cat elixir_buildpack.config | grep elixir_version | tr "=" " " | awk '{ print $2 }')
```

Comment:
As this repo has a `elixir_buildpack.config` you could just use these two lines instead and avoid the version numbers getting out of sync.